docs/run/index.md

```diff
@@ -22,20 +34,16 @@ The `starport serve` command performs the following tasks:
 - Creates accounts
 - Initializes a blockchain node
 - Starts the following processes:
-
   - Tendermint RPC
   - Cosmos SDK API
   - Faucet, optional
-
 - Watches for file changes and restarts
-
 - Exports state
 
 The `starport serve` command starts a fully operational blockchain. You can use flags to configure how the blockchain runs. All flags are optional.
```

Comment:
```suggestion
You can use flags to configure how the blockchain runs. 
```

Reply:
let's move the command description above the bullet list, and let's move "All flags are optional." to the flag description section where it will be easier to consume